### PR TITLE
Add -XX:MaxRAMPercentage parameter to SCALARDB_SERVER_OPTS

### PR DESCRIPTION
**Pull request by miseyu**
## Summary
- Added `-XX:MaxRAMPercentage` for optimizing the upper limit of `Heap Size`.
- Default -XX:MaxRAMPercentage `50.0`

## Detail
* -XX:MaxRAMPercentage `50.0`

```
resources:
  requests:
    cpu: 800m
    memory: 1Gi
  limits:
    cpu: 1200m
    memory: 1.5Gi
```
scalardb

### CODE REVIEW

File: server/Dockerfile [hunk 1]
```diff
@@ -47,9 +47,13 @@ WORKDIR /scalardb/server
 COPY log4j2.properties .
 # Support use cases where environment variables are used to configure the database
 COPY database.properties.tmpl .
+COPY docker-entrypoint.sh .
 
+ENV JAVA_OPT_MAX_RAM_PERCENTAGE '50.0'
```
Comment (brfrn169) [inline server/Dockerfile:52]: I think we can increase the default value to `70.0`? Or is there any problem if we increase it?

**Review by brfrn169 — COMMENTED** [review]
Thank you! LTGM overall. I just left a question about the percentage. Please take a look when you have time!

File: server/Dockerfile (hunk 1 above)
Comment (miseyu) [reply server/Dockerfile]: The reason for this default value is that `scalardb-server` uses a `relatively` small heap size.
Corrections have been made to this issue that you pointed out. :bow:

https://github.com/scalar-labs/scalardb/pull/392/commits/6965e64268ec3735a54307b8ec5e0c128eae9ff1

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**